app/src/main/res/layout/activity_zign_sec_authentication.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".feature.zignsec.ZignSecAuthenticationActivity">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:navigationIcon="@drawable/ic_close"
+        app:title="@string/zignsec_login_screen_title"
+        app:titleTextAppearance="?textAppearanceHeadline6"
+        tools:layout_height="?attr/actionBarSize" />
+
+    <WebView
```

Comment:
```suggestion
    <WebView
```

We have a view called `SafeWebView` which prevents potential issues when running on API 23, we might want to use that one here.

Also, we should remove the old `activity_norwegian_authentication.xml`